cli/internal/terraform/terraform.go

```diff
@@ -128,10 +128,23 @@ func (c *Client) CreateCluster(ctx context.Context) (CreateOutput, error) {
 		return CreateOutput{}, errors.New("invalid type in uid output: not a string")
 	}
 
+	var attestationURL string
+	if provider == cloudprovider.Azure {
+		attestationURLOutput, ok := tfState.Values.Outputs["attestationURL"]
+		if !ok {
+			return CreateOutput{}, errors.New("no attestationUrl output found")
+		}
+		attestationURL, ok = attestationURLOutput.Value.(string)
+		if !ok {
+			return CreateOutput{}, errors.New("invalid type in attestationUrl output: not a string")
+		}
+	}
```

Comment:
```suggestion
	var attestationURL string
	if attestationURLOutput, ok := tfState.Values.Outputs["attestationURL"]; 
		attestationURL = attestationURLOutput
	}
```
This would allow us to once again remove the CSP variable from `CreateCluster`